### PR TITLE
Added letter V language and made W section

### PR DESCRIPTION
**Pull request by viper-devil**
W and V was mixed so i made seperate section for it and added a new language in V

### CODE REVIEW

File: A-Z-checkboxes.html [hunk 1]
```diff
@@ -1651,12 +1651,15 @@ <h1>V</h1>
 <input type="checkbox" id="visual_lisp" name="pchbx" value="visual_lisp" />
 <label for="visual_lisp"> Visual LISP </label><br />
 
+<input type="checkbox" id="visual_lisp" name="pchbx" value="visual_lisp" />
+<label for="visual_lisp"> Velato </label><br />
```
Comment (chilarai) [inline A-Z-checkboxes.html:1655]: I am unfamiliar with it but can you call Visual Lisp Velato?

Comment (viper-devil) [reply A-Z-checkboxes.html:1655]: yes we can sorry i will correct it now

File: A-Z-checkboxes.html [hunk 2]
```diff
@@ -150,6 +150,9 @@ <h1>A</h1>
 
 <input type="checkbox" id="axum" name="pchbx" value="axum" />
 <label for="axum">Axum</label><br />
+
+<input type="checkbox" id="axum" name="pchbx" value="axum" />
+<label for="axum">ArnoldC</label><br />
```
Comment (chilarai) [inline A-Z-checkboxes.html:155]: Please change the label here

**Review by chilarai — APPROVED** [review]
Please do this change

**Review by chilarai — CHANGES_REQUESTED** [review]
Please do the changes as mentioned

**Review by chilarai — APPROVED** [review]
Thanks

**Pull request merged**